packages/app/components/Queue/QueueCard.tsx

```diff
@@ -0,0 +1,73 @@
+import { Card, Row, Col, Avatar, Tag } from "antd";
+import { UserOutlined } from "@ant-design/icons";
+import styled from "styled-components";
+
+const HorizontalCard = styled(Card)`
+  margin-bottom: 8px;
+  padding-left: 8px;
+  padding-right: 8px;
+`;
+
+const Text = styled.div`
+  font-size: 14px;
+  line-height: 22px;
+  color: #595959;
+`;
+
+const NameContainer = styled.div`
+  margin-left: 16px;
+`;
+
+const StatusTag = styled(Tag)`
+  width: 96px;
+  text-align: center;
+  float: right;
+  margin-right: 0;
+`;
+
+const CenterRow = styled(Row)`
+  align-items: center;
+`;
+
+interface QueueCardProps {
+  rank: number;
+  name: string;
+  questionType: string;
```

Comment:
should this be an enum/type of some type? i know we don't have concrete question types yet, but I think it's better to future proof it now

Reply:
yeah def, we made an enum in #5 but i just moved it into /common for this PR